extension/intents/tabs/tabs.js

```diff
@@ -556,3 +557,21 @@ if (!buildSettings.android) {
     },
   });
 }
+
+intentRunner.registerIntent({
+  name: "tabs.gotoTabNumber",
+  async run(context) {
+    const userInput = context.slots.tabNumber;
+    const tabNumber = findMatchingIndex(userInput);
+
+    if (tabNumber < 0) {
+      const erroMessage = userInput + " is not a vliad tab number";
+      const e = new Error(erroMessage);
```

Comment:
A couple typos here: erroMessage, and vliad.

Also, the error should always be a static string, like `new Error("Invalid tab number")`. Since the error message is our only way to categorize errors, user input in the error message will make that harder.

`displayMessage` though can contain user input. I'd recommend using something like:

```js
e.displayMessage = `"${userInput}" is not a valid number`;
```

String interpolation tends to be a bit clearer in my experience.